### PR TITLE
Fix the viewport of ViewportQuad is not applied

### LINKED ISSUE #9167: set ViewportQuad.rectangle doesn't work
Sandcastle example:

[example](https://sandcastle.cesium.com/#c=bVFda8IwFP0roS9WkHR7tpahDPYgjG2wvfQlJtc2mCYuX8WN/ffdtCqbWvKSez7uOU1klkQJPViyIBp6sgInQ0ffh1k+4cN1ZbRnUoOdTOe1jqhxHDSgZNTS4YrQCFrgnulGwX/PpQlaSN28nuD8bkbwDGIqLOsRXIbtFuyHFL69iTyBbFp/ivEZmLgOvjfWvyCSn4Mk/mi2t7KTXkZwlAmRJ4PpEBxXCNOn4WME7dfSeeTjL7Bo/AWTGcE6FfmuNcGPG+2MAqpMc2Ykn4QlT3peTfvUBUMeF0itj/1u09uh4AV/bI2Cn+k8m2Wl8wcF1agn5EF2qTIJVuWUFh66vWIeXLEJfAeecudO2crir7QUMhIpFnV28c51RrhiziGyDUq9Yb06q8oC+VdSZVh61ecIVrFDorX31XocUkrLAq+3ld4YtWH2wvkX)

After resize the window, the viewport is not applied.

out

Browser:

Google Chrome 85.0.4183.102

Operating System:

Windows 10 OS Version 1903 (Build 18362.1082)

After modify `Source/Scene/ViewportQuad.js`, it works:

```diff
  ViewportQuad.prototype.update = function(frameState) {
        if (!this.show) {
            return;
        }

        //>>includeStart('debug', pragmas.debug);
        if (!defined(this.material)) {
            throw new DeveloperError('this.material must be defined.');
        }
        if (!defined(this.rectangle)) {
            throw new DeveloperError('this.rectangle must be defined.');
        }
        //>>includeEnd('debug');

        var rs = this._rs;
        if ((!defined(rs)) || !BoundingRectangle.equals(rs.viewport, this.rectangle)) {
            this._rs = RenderState.fromCache({
                blending : BlendingState.ALPHA_BLEND,
                viewport : this.rectangle
            });
        }

        var pass = frameState.passes;
        if (pass.render) {
            var context = frameState.context;

            if (this._material !== this.material || !defined(this._overlayCommand)) {
                // Recompile shader when material changes
                this._material = this.material;

                if (defined(this._overlayCommand)) {
                    this._overlayCommand.shaderProgram.destroy();
                }

                var fs = new ShaderSource({
                    sources : [this._material.shaderSource, ViewportQuadFS]
                });
                this._overlayCommand = context.createViewportQuadCommand(fs, {
                    renderState : this._rs,
                    uniformMap : this._material._uniforms,
                    owner : this
                });
                this._overlayCommand.pass = Pass.OVERLAY;
            }

            this._material.update(context);

+           this._overlayCommand.renderState = this._rs;  // viewport may been changed
            this._overlayCommand.uniformMap = this._material._uniforms;
            frameState.commandList.push(this._overlayCommand);
        }
    };
```

It is a bug?

### PR DESCRIPTION
**Pull request by bbbbx**
fix #9167

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @bbbbx!
* :heavy_check_mark: Signed CLA found.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/Scene/ViewportQuad.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ ViewportQuad.prototype.update = function (frameState) {
 
     this._material.update(context);
 
+    this._overlayCommand.renderState = this._rs;
```
Comment (ebogo1) [inline Source/Scene/ViewportQuad.js:138]: Thanks for the fix, @bbbbx! 

@lilleyse - This change makes sense to me but I'm unable to reproduce the bugged behavior on my Linux machine. Any thoughts on this fix? I wonder if it's somehow possible to simulate a render state change in the `ViewportQuad` specs.

Comment (lilleyse) [reply Source/Scene/ViewportQuad.js:138]: Should be straightforward to test by setting the rectangle after it's been created an observing that the command's render state has the correct viewport. Something like:

```
    it("updates rectangle", function () {
      var otherRectangle = new BoundingRectangle(0, 0, 4, 4);

      scene.primitives.add(viewportQuad);
      scene.renderForSpecs();

      viewportQuad.rectangle = otherRectangle;
      scene.renderForSpecs();
      expect(scene.frameState.commandList[0].renderState.viewport).toEqual(
        otherRectangle
      );
    });
```

Comment (lilleyse) [reply Source/Scene/ViewportQuad.js:138]: Also I can confirm this PR fixes the bug.

Comment (bbbbx) [reply Source/Scene/ViewportQuad.js:138]: Hi, I have added the test, could you review it again? Thanks @lilleyse @ebogo1

File: CHANGES.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ##### Fixes :wrench:
 
 - Fixed an issue where `Camera.flyTo` would not work properly with a non-WGS84 Ellipsoid. [#9498](https://github.com/CesiumGS/cesium/pull/9498)
+- Fixed an issue that the viewport is not applied to `ViewportQuad` after setting the width and height of the rectangle.[#9511](https://github.com/CesiumGS/cesium/pull/9511)
```
Comment (ebogo1) [inline CHANGES.md:8]: @bbbbx - The spec looks good! Just a minor nitpick about wording and I think this is good to go -
```suggestion
- Fixed an issue where setting the `ViewportQuad` rectangle after after creating the viewport had no effect.[#9511](https://github.com/CesiumGS/cesium/pull/9511)
```

**Comment by ebogo1** [comment]
Thanks @bbbbx!

**Pull request merged**